apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -57,7 +57,7 @@ const Completed: FC = () => {
 
   const handleNextButtonClick = useCallback(async () => {
     const allSucceeded = await handleUpload(
-      uploadFiles.filter((file) => !file.key),
+      uploadFiles.filter((file) => file.percent === 0),
```

Comment:
Relying on the file key is not save as the key may be removed from older files when they are no longer accessible in S3.